tools/functional-tester/etcd-agent/agent.go

```diff
@@ -194,7 +194,14 @@ func archiveLogAndDataDir(log string, datadir string) error {
 		return err
 	}
 	if err := os.Rename(log, path.Join(dir, log)); err != nil {
-		return err
+		if !os.IsNotExist(err) {
+			return err
+		}
 	}
-	return os.Rename(datadir, path.Join(dir, datadir))
+	if err := os.Rename(datadir, path.Join(dir, datadir)); err != nil {
+		if !os.IsNotExist(err) {
+			return err
+		}
+	}
+	return nil
```

Comment:
after this, the non-existing files will be created by the lines after this https://github.com/coreos/etcd/pull/4516/files#diff-a56764b6877259b221f0e43b72491875R121.